api/modules/files/routes/delete.js

```diff
@@ -1,30 +1,32 @@
-var Joi = require('joi');
+"use strict";
 
-var errors = require('../../../classes/errors');
-var prereq = require('../../../classes/prerequisites');
+const Joi = require(`joi`);
 
-var controller = require('../controller');
-var Model = require('../model');
+const Errors = require(`../../../classes/errors`);
+const Prerequistes = require(`../../../classes/prerequisites`);
+
+const FilesModel = require(`../model`);
+const filesController = require(`../controller`);
 
 module.exports = [{
-  method: 'DELETE',
-  path: '/files/{id}',
+  method: `DELETE`,
+  path: `/files/{id}`,
   config: {
     pre: [
-      prereq.confirmRecordExists(Model, {
-        mode: 'param',
-        requestKey: 'id'
+      Prerequistes.confirmRecordExists(FilesModel, {
+        mode: `param`,
+        requestKey: `id`
       }),
-      prereq.validateUser(),
-      prereq.validateOwnership()
+      Prerequistes.validateUser(),
+      Prerequistes.validateOwnership()
     ],
-    handler: controller.delete.bind(controller),
-    description: 'Delete a single file object based on `id`.',
+    handler: filesController.delete.bind(filesController),
```

Comment:
`handler: args => filesController.delete(...args),` although it feels odd that this can't just be a static function and needs explicit tying to a specific fileController instance. It suggests we might have unnecessary data capturing in the fileController class.